db/models.py

```diff
@@ -1 +1,43 @@
 from django.db import models
+
+
+class Race(models.Model):
+    race_to_choose = (
+        ("Elf", "Elf"),
+        ("Dwarf", "Dwarf"),
+        ("Human", "Human"),
+        ("Ork", "Ork")
+    )
+    name = models.CharField(
+        max_length=255,
+        unique=True,
+        choices=race_to_choose
+    )
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(
+        max_length=255,
+        unique=True
+    )
+    bonus = models.CharField(
+        max_length=255
```

Comment:
Consider using a more appropriate field type for the `bonus` attribute in the `Skill` model. If `bonus` is meant to store numerical values, a `DecimalField` or `IntegerField` might be more suitable than `CharField`.